src/models/gaze17/repairs.md

```diff
@@ -0,0 +1,205 @@
+# Gazelle (Parts & Repairs)
+
+Many components in your Gazelle can be upgraded or replaced as necessary. Follow these step-by-step guides for instructions:
+
+- [Removing the bottom cover](#removing-the-bottom-cover)
+- [Replacing the RAM](#replacing-the-ram)
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the battery](#replacing-the-battery)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the keyboard](#replacing-the-keyboard)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 5 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 13 bottom panel screws.
+
+![Bottom panel screws](./img/bottom-panel-screws.webp)
+
+3. Pull the bottom panel off, starting from the hinges in the back.
+
+## Replacing the RAM:
+
+The Gazelle 16 supports up to 64GB (2x32GB) of DDR4 SO-DIMMs running at 3200MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
```

Comment:
s/Gazelle 16/Gazelle 17

Reply:
Of course. Fixed with dabd0ecbacdd840e8dcfe1f78d7d9e4262653a94.